python/hail/methods/impex.py

```diff
@@ -1145,6 +1147,9 @@ def import_table(paths,
     skip_blank_lines : :obj:`bool`
         If ``True``, ignore empty lines. Otherwise, throw an error if an empty
         line is found.
+    force_bgz : :obj:`bool`
+        If ``True``, load files as blocked gzip files, assuming
+        that they were actually compressed using the BGZ codec.
```

Comment:
I think the end of the notes should give some context for this new parameter, similar but briefer than is done in `import_vcf`. In particular, that block gzip files files ending in `.bgz` will be decompressed in parallel, that this can be forced for block gzip files with the wrong extension using the parameter. With bgz, do we get a partition per block? How does this interact with min_partitions?

Reply:
It isn't a partition per block. Has nothing to do with min_partitions.